### PR TITLE
feat: Updating road map

### PR DESCRIPTION
**Pull request by JanCizmar**

### CODE REVIEW

**Review by stepan662 — APPROVED** [review]

File: blog/2022-02-07-updating-road-map.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+title: Updating Roadmap
+
+authors: [jcizmar]
+
+image: "/img/blog/avatar/ogImage.png"
+
+tags: [roadmap, tolgee, future]
+---
+
+In Q4 of 2021 we planned to finish features like translation memory, automated/machine translating, glossaries or paywall
```
Comment (marketachalupnikova) [inline blog/2022-02-07-updating-road-map.md:13]: 2021,

File: blog/2022-02-07-updating-road-map.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+authors: [jcizmar]
+
+image: "/img/blog/avatar/ogImage.png"
+
+tags: [roadmap, tolgee, future]
+---
+
+In Q4 of 2021 we planned to finish features like translation memory, automated/machine translating, glossaries or paywall 
+for Tolgee cloud. However, we spent a lot of time on upgrading a website and improving JS integrations. Sadly, we 
+haven't finished any of the features, we wanted to finish before end of the year. So we have to reflect it in our
```
Comment (marketachalupnikova) [inline blog/2022-02-07-updating-road-map.md:15]: the end of the year

File: blog/2022-02-07-updating-road-map.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+roadmap. We decided to reschedule everything and target more on the segment of developers than translators.
+Ladies and gentlemen, we are introducing [improved roadmap](/roadmap)! 🎉
+
+## 2022/Q1
+We are just finishing the translation memory, machine translations and automated translating. So users translating their
+projects in Tolgee platform will feel much more comfortable. The platform will be able to suggest translations from 
+the memory and from providers like Goolge Translate or AWS translate.
```
Comment (marketachalupnikova) [inline blog/2022-02-07-updating-road-map.md:22]: Google

File: blog/2022-02-07-updating-road-map.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+We would also like to provide more export options, create nice project dashboard with statistics, enable users to see
```
Comment (marketachalupnikova) [inline blog/2022-02-07-updating-road-map.md:24]: a nice

File: blog/2022-02-07-updating-road-map.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+## 2022/Q1
+We are just finishing the translation memory, machine translations and automated translating. So users translating their
+projects in Tolgee platform will feel much more comfortable. The platform will be able to suggest translations from 
+the memory and from providers like Goolge Translate or AWS translate.
+
+We would also like to provide more export options, create nice project dashboard with statistics, enable users to see 
+history of translations or add more permission options.
+
+## 2022/Q2
+In the second quarter of 2022 we would like to introduce Glossaries, so user's will be able to manage their terminology
```
Comment (marketachalupnikova) [inline blog/2022-02-07-updating-road-map.md:28]: 2022,

Comment (marketachalupnikova) [reply blog/2022-02-07-updating-road-map.md:28]: users

File: blog/2022-02-07-updating-road-map.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+projects in Tolgee platform will feel much more comfortable. The platform will be able to suggest translations from 
+the memory and from providers like Goolge Translate or AWS translate.
+
+We would also like to provide more export options, create nice project dashboard with statistics, enable users to see 
+history of translations or add more permission options.
+
+## 2022/Q2
+In the second quarter of 2022 we would like to introduce Glossaries, so user's will be able to manage their terminology 
+and reuse that across multiple projects as well as we would also like to introduce a billing for Tolgee Cloud, so user 
+would be able to buy seats for their team members.
```
Comment (marketachalupnikova) [inline blog/2022-02-07-updating-road-map.md:30]: this paragraph is just one very long phrase. Rewrite it to two shorter phrases so its easier to read and understand.

File: blog/2022-02-07-updating-road-map.md [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+
+## 2022/Q3-Q4
+In the rest of the year, we would like to provide QA checks, design tools integrations, support for mobile apps, QA checks,
+Git integrations, branching or translator marketplace.
+
+## Full roadmap
+Since we moved to GitHub, you can check progress of every upcoming feature in the project dashboard. Or by clicking
```
Comment (marketachalupnikova) [inline blog/2022-02-07-updating-road-map.md:37]: the progress

File: src/pages/roadmap/index.tsx [hunk 8]
```diff
@@ -54,36 +54,68 @@ than finding the ideal translator for your app.`,
...
-services like DeepL, Google Translate, or AWS translate will help you to 
-translate your strings automatically.`,
+services like DeepL, Google Translate, or AWS translate will help by suggesting or 
+translating your new keys automatically when enabled.`,
+        inProgressLink: "https://github.com/tolgee/server/pull/920"
+      },
+      {
+        name: "Translating on production",
+        description: `Currently, in-context localization works only when application is in development mode which is cool, but we would like
+         to enable users to translate directly in the deployed app by providing API key using the Tolgee Tools chrome plugin.`,
```
Comment (marketachalupnikova) [inline src/pages/roadmap/index.tsx:76]: tool Chrome

File: src/pages/roadmap/index.tsx [hunk 9]
```diff
@@ -54,36 +54,68 @@ than finding the ideal translator for your app.`,
...
         description: `Don't translate everything manually. With an automated translations feature, 
-services like DeepL, Google Translate, or AWS translate will help you to 
-translate your strings automatically.`,
+services like DeepL, Google Translate, or AWS translate will help by suggesting or 
+translating your new keys automatically when enabled.`,
+        inProgressLink: "https://github.com/tolgee/server/pull/920"
+      },
+      {
+        name: "Translating on production",
+        description: `Currently, in-context localization works only when application is in development mode which is cool, but we would like
```
Comment (marketachalupnikova) [inline src/pages/roadmap/index.tsx:75]: the application

File: src/pages/roadmap/index.tsx [hunk 10]
```diff
@@ -54,36 +54,68 @@ than finding the ideal translator for your app.`,
...
+         to enable users to translate directly in the deployed app by providing API key using the Tolgee Tools chrome plugin.`,
+        inProgressLink: "https://github.com/tolgee/tolgee-js/pull/1775"
+      },
+      {
+        name: "More export options",
+        description: `More formats (json, xliff) and export options like filtering or structuring will be provided, so exports will be more versatile.`
+      },
+      {
+        name: "Project Dashboard",
+        description: `To see detailed statistic for the project and for each language.`
```
Comment (marketachalupnikova) [inline src/pages/roadmap/index.tsx:85]: statistics

**Pull request merged**